app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,9 @@
+class GuestRoomsController < ApplicationController
+
+  def create
+   GuestRoom.create!(guest_id: params[:id], room_id: params[:room_id])
```

Comment:
This is totally fine for an IC, but in projects and in the future, try to use `create!` only in tests or seeds files where we don't care if exceptions are thrown if the data passed in is wrong. In controllers, we want to make sure users have a more friendly experience so we want just `create` (or on a project you might do some error handling with flash messages)